src/utility-types.ts

```diff
@@ -37,3 +38,21 @@ export interface MessageValue<
    */
   readonly parent?: MainWorker
 }
+
+/**
+ * TODO: Needs description.
+ *
+ * @template Worker Type of worker.
+ * @template Response Type of response of execution. This can only be serializable data.
+ */
+export interface ThisNeedsAName<Worker extends IWorker, Response = unknown> {
```

Comment:
@jerome-benoit @pioardi We need a name for this :thinking:

Reply:
PromiseResponseWrapper ? Mmmm